File: backend/voting-app/src/main/java/com/tu/votingapp/dto/response/referendum/ReferendumResponseDTO.java
package com.tu.votingapp.dto.response.referendum;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ReferendumResponseDTO {
    private Long id;
    private String title;
    private String description;
    private String question;
    private LocalDate startDate;
    private LocalDate endDate;
    private String status;
    private Long createdById;
    private List<OptionResponseDTO> options;
}

File: frontend/android/app/src/main/java/com/example/android/dto/response/referendum/ReferendumResponseDTO.kt
package com.example.android.dto.response.referendum

import kotlinx.serialization.Serializable

@Serializable
data class ReferendumResponseDTO(
    val id: Long,
    val title: String,
    val description: String?,
    val question: String,
    val startDate: String,
    val endDate: String,
    val status: String,
    val createdById: Long,
    val options: List<OptionResponseDTO>
)